Migrate SQL docs (#521)
redpanda-data/cloud-docs: modules/reference/pages/sql/sql-functions/timestamp-functions/timestamp-micros.adoc

= TIMESTAMP_MICROS
:description: The TIMESTAMP_MICROS() function converts a given UNIX timestamp value in microseconds since 1970-01-01 00:00:00 UTC into a timestamp.
:page-topic-type: reference

> The `TIMESTAMP_MICROS()` function converts a given UNIX timestamp value in microseconds since 1970-01-01 00:00:00 UTC into a timestamp. Its syntax is as follows:

Reviewer: ```suggestion
The `TIMESTAMP_MICROS()` function converts a given UNIX timestamp value in microseconds since 1970-01-01 00:00:00 UTC into a timestamp. Its syntax is:
```

[source,sql]
----
SELECT TIMESTAMP_MICROS(BIGINT)
----

Its input type is a `BIGINT` expression representing a UNIX timestamp in microseconds and the return data type is a timestamp.

== Examples

=== Basic `TIMESTAMP_MICROS()` function

The below example uses the `TIMESTAMP_MICROS()` function to convert a given UNIX timestamp in microseconds into a timestamp without a timezone:

[source,sql]
----
SELECT TIMESTAMP_MICROS(2280419000000000) AS timestamp_microsvalues;
----

Output:

[source,sql]
----
+-----------------------------+
| timestamp_microsvalues      |
+-----------------------------+
| 2042-04-06 17:43:20         |
+-----------------------------+
----

=== `TIMESTAMP_MICROS()` function using columns

> Suppose you have a table named **timemicro_example** with the following UNIX time values in microseconds in the *unix_timestamp* column:

Reviewer: ```suggestion
Suppose a table named **timemicro_example** has the following UNIX time values in microseconds in the *unix_timestamp* column:
```
